mllib/src/main/scala/org/apache/spark/ml/classification/ProbabilisticClassifier.scala

```diff
@@ -230,21 +230,19 @@ private[ml] object ProbabilisticClassificationModel {
    * Normalize a vector of raw predictions to be a multinomial probability vector, in place.
    *
    * The input raw predictions should be nonnegative.
-   * The output vector sums to 1, unless the input vector is all-0 (in which case the output is
-   * all-0 too).
+   * The output vector sums to 1, when the input vector is all-0, exception will be thrown.
```

Comment:
This isn't grammatical. You might just move the new note to a proper throws tag in the scaladoc here anyway

Reply:
Updated. Thanks!